### PR TITLE
fix: create-order-issue

### PR DESCRIPTION
**Pull request by leonthaibao**
This PR fixed the create_order issue (didn't use useEffect hook correctly)
Skip loading toast when adding items
Fix the invalid link issue and deprecated props when migrating to Next.js version 13. Detail is [here](**[url](https://nextjs.org/docs/messages/invalid-new-link-with-extra-anchor)**)

### CODE REVIEW

File: pages/checkout/index.js [hunk 1]
```diff
@@ -26,10 +30,11 @@ const Checkout = () => {
...
       if (secondsToGo === 0) {
         clearInterval(timer);
-        message.error('Out of time, please try again!').then(r => router.push('/cart'));
-
+        message
+          .error("Out of time, please try again!")
+          .then((r) => router.push("/cart"));
       }
-    },  1000) // interval check after 1s for test purpose
+    }, 1000); // interval check after 1s for test purpose
```
Comment (anhldbk) [inline pages/checkout/index.js]: @leonthaibao pls update the comment

File: components/ProductCard.js [hunk 2]
```diff
@@ -29,63 +26,59 @@ const ProductCard = props => {
   useEffect(() => {
     if (firstRun.current) {
       firstRun.current = false;
+      toast.success(`${props.name} added`, {
+        id: "test",
```
Comment (anhldbk) [inline components/ProductCard.js]: @leonthaibao do we need this?

Comment (leonthaibao) [reply components/ProductCard.js]: I've removed these, pls re-check it

File: pages/checkout/index.js (hunk 1 above)
Comment (leonthaibao) [reply pages/checkout/index.js]: I've updated these, pls re-check it

**Comment by anhldbk** [comment]
@leonthaibao LGTM. Thank you!

**Pull request merged**